packages/lib/ren-tx/src/configs/genericMint.ts

```diff
@@ -41,6 +41,17 @@ const lockChainMap = {
 };
 */
 
+const hexify = (obj: { [key: string]: any }) => {
```

Comment:
It would be good to comment this or make the name a bit more explanatory (e.g. `iterativeBufferToHex`).